### PR TITLE
Add instructions for installing from packages

### PR DESCRIPTION
**Pull request by philippem**

### CODE REVIEW

File: elements-code-tutorial/installing-elements.md [hunk 1]
```diff
@@ -8,6 +8,86 @@ permalink: /elements-code-tutorial/installing-elements
...
+
+| Platform | Package Extension |
+| -------- | --------  |
+| Linux 64-bit on Intel or AMD | `-x86_64-linux-gnu.tar.gz` |
+| macOS (Intel and Apple Silion) | `-osx-64.tar.gz` |
+| Windows 64-bit | `-win64.zip` |
+| 64-bit Raspberry Pi and 64 bit ARM Linux | `-aarch64-linux-gnu.tar.gz` |
+
+
+Note that if you are following this tutorial on a Windows computer, you will need to have a version of bash for Windows (e.g. git for windows).
```
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md:33]: git for windows is a version of git that includes a full bash shell, it's pretty slick. but people may not be familiar with it here. should we put less? as the entire tutorial is based on bash, windows users won't be doing it out of the box

https://gitforwindows.org/

File: elements-code-tutorial/installing-elements.md [hunk 2]
```diff
@@ -8,6 +8,86 @@ permalink: /elements-code-tutorial/installing-elements
...
+gpg:                 aka "Steven Roose (Jabber) <steven@konuro.net>" [unknown]
+gpg: WARNING: This key is not certified with a trusted signature!
+gpg:          There is no indication that the signature belongs to the owner.
+Primary key fingerprint: 8CC9 74D9 CFD0 34DC EED2  13B0 2A57 E0A6 10D7 F19C
+     Subkey fingerprint: DE10 E826 29A8 CAD5 5B70  0B97 2F2A 88D7 F8D6 8E87
+```
+
+Extract the archive, and copy the `elementsd` and `elements-cli` executables to `/usr/local/bin`.
+
+Check that you can run the packages, using the `-version` option:
```
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md:64]: I did not have to from the Linux tarball.

File: elements-code-tutorial/installing-elements.md [hunk 3]
```diff
@@ -8,6 +8,86 @@ permalink: /elements-code-tutorial/installing-elements
...
+Elements Core RPC client version elements-0.21.0.2
+```
+
+You are now ready to proceed with the tutorial.
+
+
+## Building Elements from Source
+
+
 As we will be building Elements from the source code, we first need to pull the code from the [GitHub repository](https://github.com/elementsproject/elements) where it is maintained. We'll use "git" for this. You can ignore installation steps within this tutorial for software that you already have. If you do not have git installed (you can check by running the ``git --version`` command) you can install it using:
```
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md:91]: yes let's move it to another section

File: elements-code-tutorial/installing-elements.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md:33]: will remove the reference to git for windows.

File: elements-code-tutorial/installing-elements.md [hunk 4]
```diff
@@ -8,6 +8,86 @@ permalink: /elements-code-tutorial/installing-elements
...
+
+Extract the archive, and copy the `elementsd` and `elements-cli` executables to `/usr/local/bin`.
+
+Check that you can run the packages, using the `-version` option:
+
+
+```
+$ elementsd -version
+Elements Core version elements-0.21.0.2
+Copyright (C) 2009-2021 The Elements Project developers
```
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md:70]: updated in later version

File: _includes/header.html [hunk 5]
```diff
@@ -55,8 +55,10 @@
             <li role="separator" class="divider"></li>
             <li><a href="/elements-code-tutorial/easy-run-code">An easy way to run the main tutorial code</a></li>
             <li><a href="/elements-code-tutorial/advanced-examples">Advanced examples</a></li>
+            <li><a href="/elements-code-tutorial/building-elements-from-source">Building Elements from source</a></li>	    
             <li role="separator" class="divider"></li>
-            <li><a href="https://github.com/ElementsProject/elements">Elements on Github</a></li>
```
Comment (wintercooled) [inline _includes/header.html:59]: If you remove this then the last line in the menu is a line separator - so remove the separator above as well looks neater or else you get a double end to the menu.

File: elements-code-tutorial/installing-elements.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (wintercooled) [reply elements-code-tutorial/installing-elements.md:79]: I see you have added a new page - but this (line 75 to 148 maybe) then needs removing to complete the 'move'.

File: _includes/header.html (hunk 5 above)
Comment (philippem) [reply _includes/header.html:59]: did not mean to delete that, will restore.

File: elements-code-tutorial/installing-elements.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (philippem) [reply elements-code-tutorial/installing-elements.md]: done

File: elements-code-tutorial/installing-elements.md [hunk 6]
```diff
@@ -8,75 +8,70 @@ permalink: /elements-code-tutorial/installing-elements
...
 
-~~~~
-cd
-cd elements
-./autogen.sh
-./configure LDFLAGS="-L/home/yourusername/bdb4/lib/" CPPFLAGS="-I/home/yourusername/bdb4/include/"
-make
-sudo make install
-~~~~
+Extract the archive, and copy the `elementsd` and `elements-cli` executables to `/usr/local/bin`.
```
Comment (wintercooled) [inline elements-code-tutorial/installing-elements.md:59]: Might as well add elements-qt as well

**Comment by wintercooled** [comment]
Hi Phil.

In order to now use the binaries - we need to change any uses of alias to also point to the binaries. So this...

```
alias e1-dae="$HOME/elements/src/elementsd -datadir=$HOME/elementsdir1"
alias e1-cli="$HOME/elements/src/elements-cli -datadir=$HOME/elementsdir1"
alias e2-dae="$HOME/elements/src/elementsd -datadir=$HOME/elementsdir2"
alias e2-cli="$HOME/elements/src/elements-cli -datadir=$HOME/elementsdir2"
alias e1-qt="$HOME/elements/src/qt/elements-qt -datadir=$HOME/elementsdir1"
alias e2-qt="$HOME/elements/src/qt/elements-qt -datadir=$HOME/elementsdir2"
```

Would become...

```
alias e1-dae="elementsd -datadir=$HOME/elementsdir1"
alias e1-cli="elements-cli -datadir=$HOME/elementsdir1"
alias e2-dae="elementsd -datadir=$HOME/elementsdir2"
alias e2-cli="elements-cli -datadir=$HOME/elementsdir2"
alias e1-qt="elements-qt -datadir=$HOME/elementsdir1"
alias e2-qt="elements-qt -datadir=$HOME/elementsdir2"
```

And there are other instances of the use of alias within the project - the easy run examples for example.

File: elements-code-tutorial/installing-elements.md (hunk 6 above)
Comment (philippem) [reply elements-code-tutorial/installing-elements.md]: added

**Pull request merged**